cycle_en_terre_website_design/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# from . import controllers
+# from . import models
+# from . import wizard
```

Comment:
This file should be empty because you don't have neither controllers nor models nor wizard. It can seem strange to have an empty file, but it must exist for odoo and python to understood that this directory is a module. :)
By the way, don't put licences in the `__init__.py` files as they don't contain any real code. :)

Reply:
Done